src/Illuminate/Database/Query/Builder.php

```
@@ -522,6 +522,32 @@ public function orWhere($column, $operator = null, $value = null)
     }
 
     /**
+     * Adds multiple where clauses to the query
+     *
+     * @param array $conditions
+     * @return $this
+     */
+    public function whereAll($conditions)
+    {
+        foreach ($conditions as $condition) {
+            if (sizeof($condition) == 2) {
+                @list($column, $value) = $condition;
+            } else {
+                @list($column, $operator, $value, $clause) = $condition;
```

bad idea to silence errors